connectors/sources/tests/fixtures/oracle/requirements.txt

```
@@ -1 +1 @@
-cx-Oracle==7.3.0
+oracledb
```

```suggestion
oracledb==1.2.2
```

E2E-test should use the same version as the framework.